client-src/elements/chromedash-activity-prevent_toast.js

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
It doesn't seem like this component is used anywhere. I cannot tell what the intention is to avoid the initial toast being displayed for the short time window at the beginning of a new load. What do you see when you run the project locally?